rand_distr/src/poisson.rs

```diff
@@ -16,8 +16,19 @@ use rand::Rng;
 
 /// The Poisson distribution `Poisson(lambda)`.
 ///
+/// The Poisson distribution is a discrete probability distribution with
+/// rate parameter `λ`. It models the number of events occurring in a fixed
+/// interval of time or space.
+///
 /// This distribution has a density function:
-/// `f(k) = lambda^k * exp(-lambda) / k!` for `k >= 0`.
+/// `f(k) = λ^k * exp(-λ) / k!` for `k >= 0`.
+///
+/// # Plot
+///
+/// The following plot shows the Poisson distribution with various values of `λ`.
+/// Note how a higher `λ` results in more expected events per unit interval.
```

Comment:
I prefer this wording to avoid confusion with results (outcomes) of the distribution:

Note how the expected number of events increases with `λ`.